### PR TITLE
feature/BNGP-4896-creditpurchase-focus-control

### PR DESCRIPTION
**Pull request by soorajvdefra**

### CODE REVIEW

**Review by KiranVarma02 — COMMENTED** [review]
Would normal approach of focus which is done in other places not work here?

File: packages/webapp/src/views/credits-purchase/check-and-submit.html [hunk 1]
```diff
@@ -57,7 +57,7 @@
     'value': credits.amounts.a1,
     'valueDataTestId': 'credits-a1-value',
     'valueId': 'credits-a1-value',
-    'href': credits.changeUrl,
+    'href': credits.changeUrl + '?focus=a1-units',
```
Comment (KiranVarma02) [inline packages/webapp/src/views/credits-purchase/check-and-submit.html:60]: I have just tested and this works perfect without any js. Please try and let me know

` 'href': credits.changeUrl+'#a1-units',`

File: packages/webapp/src/views/credits-purchase/check-and-submit.html [hunk 2]
```diff
@@ -66,7 +66,7 @@
     'value': credits.amounts.a2,
     'valueDataTestId': 'credits-a2-value',
     'valueId': 'credits-a2-value',
-    'href': credits.changeUrl,
+    'href': credits.changeUrl + '?focus=a2-units',
```
Comment (KiranVarma02) [inline packages/webapp/src/views/credits-purchase/check-and-submit.html:69]: `'href': credits.changeUrl+'#a2-units',`

**Review by KiranVarma02 — CHANGES_REQUESTED** [review]

File: packages/webapp/src/views/credits-purchase/check-and-submit.html (hunk 1 above)
Comment (soorajvdefra) [reply packages/webapp/src/views/credits-purchase/check-and-submit.html:60]: Yes, Kiran, your suggestion should work. However, there's another issue: using 'href': credits.changeUrl+'#a1-units' will focus on the element with scrolling. If the user needs to confirm which box it is, they will need to scroll up!. I will demonstrate this tomorrow.

**Review by KiranVarma02 — APPROVED** [review]

**Review by KiranVarma02 — APPROVED** [review]

**Comment by shaundefra** [comment]
tested locally

**Pull request merged**